tests/data/test_dataloader.py

```diff
@@ -735,8 +741,10 @@ def test_token_counting_func_dataloader_setting(
             },
             **common_args
         })
+        ds_mock = MagicMock()
```

Comment:
Why is this mocked but not for streaming-finetuning?

Reply:
oh, nvm it is!

Reply:
I assume we need the mock here because the streamingtextdataset has attributes that are accessed.